### PR TITLE
ADR-1030 Add OtherMaltedGrains spirits page

### PR DESCRIPTION
**Pull request by davidfes**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -556,6 +557,23 @@ ethyleneGasUsed.change.hidden = Ethylene Gas used
...
 
+otherMaltedGrains.title = Tell us about the other malted grains you have used
+otherMaltedGrains.heading = Tell us about the other malted grains you have used
+otherMaltedGrains.subheading1 = What other malted grains have you used?
+otherMaltedGrains.subheading2 = How much have you used?
+otherMaltedGrains.hint = Enter weight in tonnes, up to 2 decimal places.
+otherMaltedGrains.checkYourAnswersLabel = OtherMaltedGrains
+otherMaltedGrains.error.otherMaltedGrainsTypes.required = TBD Enter otherMaltedGrainsTypes
+otherMaltedGrains.error.otherMaltedGrainsQuantity.required = Enter how much other malted grains you have used.
+otherMaltedGrains.error.otherMaltedGrainsTypes.length = TBD must be 120 characters or less
```
Comment (sini-george) [inline conf/messages.en:568]: The length ones can be taken out as they are not used

Comment (davidfes) [reply conf/messages.en]: Fixed

File: app/views/spiritsQuestions/OtherMaltedGrainsView.scala.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        )
+
+        @pageHeading(messages("otherMaltedGrains.heading"))
+
+        @subHeading(messages("otherMaltedGrains.subheading1"))
+
+        @govukInput(
+            InputViewModel(
+                field = form("otherMaltedGrainsTypes"),
+                label = LabelViewModel(messages("otherMaltedGrains.subheading1")).asVisuallyHidden()
```
Comment (yasaracar) [inline app/views/spiritsQuestions/OtherMaltedGrainsView.scala.html:53]: Testers raised an issue with the subheadings. Sinimol did a fix on a parallel branch where she uses the label instead but with some extra css. Can you use the same fix here please? 
[ADR-988 Pull Request](https://github.com/hmrc/alcohol-duty-returns-frontend/pull/77/files#diff-405601217f4e953299b66ced2b67313a2168a99b5055710c682f37ee5a718a6bR51)

File: app/views/spiritsQuestions/OtherMaltedGrainsView.scala.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            )
+            .withWidth(ThreeQuarters)
+        )
+
+        @subHeading(messages("otherMaltedGrains.subheading2"))
+
+        @govukInput(
+            InputViewModel(
+                field = form("otherMaltedGrainsQuantity"),
+                label = LabelViewModel(messages("otherMaltedGrains.subheading2")).asVisuallyHidden()
```
Comment (yasaracar) [inline app/views/spiritsQuestions/OtherMaltedGrainsView.scala.html:63]: same as above

**Review by yasaracar — CHANGES_REQUESTED** [review]

File: app/views/spiritsQuestions/OtherMaltedGrainsView.scala.html (hunk 3 above)
Comment (davidfes) [reply app/views/spiritsQuestions/OtherMaltedGrainsView.scala.html:63]: fixed here too

**Review by yasaracar — APPROVED** [review]

**Review by yasaracar — APPROVED** [review]

**Pull request merged**